InnoClubs/InnoClubs/apps/authorization/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index')
+]
```

Comment:
```suggestion
]

```
Гитхаб указывает, что это не оч + в [интернете](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) пишут, что это плохой стиль не ставить newline в конце файла